packages/design-tokens/docs/StoryBoard.stories.tsx

```diff
@@ -471,15 +466,15 @@ export const Everything: Story = () => {
     <div
       style={{
         height: "100vh",
-        backgroundColor: theme.color.gray["100"],
+        backgroundColor: 'theme.color.gray["100"]',
       }}
     >
       <div
         style={{
           position: "fixed",
-          zIndex: 10,
-          right: theme.spacing.xxl,
-          top: theme.spacing.md,
+          zIndex: "10",
+          right: "theme.spacing.xxl",
```

Comment:
I don't think these should be strings 👀

Reply:
Hmm, this got auto formatted for some reason